src/endpoints/NewsroomCategories/types.ts

```diff
@@ -18,10 +20,12 @@ export interface CreateRequest {
             description?: string;
         };
     };
+    image?: UploadedImage | null;
```

Comment:
Hey @pestaa, I think this should be a `string | null`? 🤔 I've tried providing the `UploadedImage` directly but